e2e-tests/tests/e2e_flows.py

```diff
@@ -0,0 +1,310 @@
+import pytest
+import json
+
+from common.assertions import equal_json_strings
+from common.methods import analyze, anonymize, analyzer_supported_entities
+
+
+@pytest.mark.api
+def test_given_a_correct_analyze_input_then_anonymize_success():
+    analyzer_request = json.loads("""
+    {
+        "text": "John Smith drivers license is AC432223",
+        "language": "en"
+    }
+    """)
+
+    response_status, response_content = analyze(json.dumps(analyzer_request))
+
+    expected_response = """
+    [
+        {"entity_type": "PERSON", "start": 0, "end": 10, "score": 0.85, 
+        "analysis_explanation": {
+            "recognizer": "SpacyRecognizer", "pattern_name": null, "pattern": null, "original_score": 0.85, 
+            "score": 0.85, "textual_explanation": "Identified as PERSON by Spacy's Named Entity Recognition", 
+            "score_context_improvement": 0, "supportive_context_word": "", "validation_result": null 
+            }
+        },
+        {"entity_type": "US_DRIVER_LICENSE", "start": 30, "end": 38, "score": 0.6499999999999999, 
+        "analysis_explanation": {
+            "recognizer": "UsLicenseRecognizer", "pattern_name": "Driver License - Alphanumeric (weak)", 
+            "pattern": "\\\\b([A-Z][0-9]{3,6}|[A-Z][0-9]{5,9}|[A-Z][0-9]{6,8}|[A-Z][0-9]{4,8}|[A-Z][0-9]{9,11}|[A-Z]{1,2}[0-9]{5,6}|H[0-9]{8}|V[0-9]{6}|X[0-9]{8}|A-Z]{2}[0-9]{2,5}|[A-Z]{2}[0-9]{3,7}|[0-9]{2}[A-Z]{3}[0-9]{5,6}|[A-Z][0-9]{13,14}|[A-Z][0-9]{18}|[A-Z][0-9]{6}R|[A-Z][0-9]{9}|[A-Z][0-9]{1,12}|[0-9]{9}[A-Z]|[A-Z]{2}[0-9]{6}[A-Z]|[0-9]{8}[A-Z]{2}|[0-9]{3}[A-Z]{2}[0-9]{4}|[A-Z][0-9][A-Z][0-9][A-Z]|[0-9]{7,8}[A-Z])\\\\b", 
+            "original_score": 0.3, "score": 0.6499999999999999, "textual_explanation": null, 
+            "score_context_improvement": 0.3499999999999999, "supportive_context_word": "driver", 
+            "validation_result": null 
+            }
+        }
+    ]
+    """
+    assert response_status == 200
```

Comment:
IMO This one applies to all tests:
Since the test is very long, I would split this into two methods. something like:
```
def test_given_text_with_PII_then_analyze_and_anonymize_succeed():
      analyzer_request = json.loads ....
      analyzer_data = analyze_and_check_result(analyzer_request, expected_response)
      anonymizer_request = { ....
      anonymize_and_check_Result(anonymizer_request, expected_response)
```

Reading the script will be much easier. Debug and changes will be easier to follow.
By creating the methods: 
1. analyze_and_check_result
2. anonymize_and_check_Result
It will be easier to add more tests in the future (if needed).